src/website/apps/api/v2/viewsets/career.py

```
@@ -28,14 +26,27 @@ def get_serializer_class(self):  # type: ignore[override]
 
 class CareerViewSet(SlugModelViewSetMixin, OptimizedQuerySetMixin, viewsets.ReadOnlyModelViewSet):
     queryset = Career.objects.all()
-    permission_classes = [OpenAPIPermission]
     pagination_class = StandardPageNumberPagination
     filter_backends = [DjangoFilterBackend, SearchFilter, OrderingFilter]
     filterset_class = CareerFilterSet
     search_fields = ['title', 'type']
     ordering_fields = ['id', 'title', 'closing_date', 'created', 'modified']
     ordering = ['-closing_date', 'title']
     select_related_fields = ['department']
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)